cmake/TestImpactFramework/LYTestImpactFramework.cmake

```diff
@@ -630,7 +619,8 @@ endfunction()
 
 #! ly_test_impact_post_step: runs the post steps to be executed after all other cmake scripts have been executed.
 function(ly_test_impact_post_step)
-    if(NOT LY_TEST_IMPACT_ACTIVE)
+    if(NOT O3DE_TEST_IMPACT_ACTIVE)
+        message("TIAF is disabled, no post step will be performed.")
```

Comment:
It seems like there should be a message for when TIAF is active.
For normal users, this message would output every time during CMake configure.
Also this message seems like it should be at the [DEBUG](https://cmake.org/cmake/help/latest/command/message.html?highlight=message) log level, since it is seems internal to the Test Impact Analysis Framework impl

Reply:
Noted 👍